knapsack: split introduction and instructions (#2440)
exercism/problem-specifications: exercises/knapsack/instructions.md

> # Description

Author: ```suggestion
# Instructions
```

> Your task is to determine which items to take to maximize their value, taking into account the knapsack's carrying capacity.

Reviewer: The "Your task" line was at the bottom of the instructions in a bunch of the other reviews. Should we try to be consistent about putting the "ask" at the top/bottom?

Author: It should be at the top.

> Your task is to determine which items to take to maximize their value, taking into account the knapsack's carrying capacity.

Author: ```suggestion

Your task is to determine which items to take so that the total value of his selection is maximized, taking into account the knapsack's carrying capacity.
```

# Instructions

> 

Author: ```suggestion
```
